python/dolfinx/__init__.py

```diff
@@ -6,9 +6,14 @@
 """Main module for DOLFINx"""
 
 # flake8: noqa
-
+import os
 import sys
 
+if sys.platform.startswith("win32"):
```

Comment:
Hardwired paths don't look great.

Reply:
This has to be removed completely - on Windows the CI needs to manually inject these paths at install time (python on windows will not respect PATH for dlls)

Reply:
conda Windows builds are now working (Python build finishes, cpp tests pass), but Python is stopping here because these directories don't exist.

If these came from the env like `DOLFINX_ADD_DLL_DIRECTORY_PATH` and/or checked if they existed before adding them, it would be easier.